Documentation/fleet-k8s-compared.md

```diff
@@ -0,0 +1,32 @@
+# fleet Compared to Kubernetes
+
+This table briefly compares key features of `fleet` to those in Kubernetes:
+
+|Feature|Description|fleet|kubernetes|
+|------------|-----------|-----|----------|
+| Load balancer | Kubernetes has built-in *services* abstraction. Fleet doesn't provide this feature directly, but it can [deploy a stacked solution][service-discovery] built atop app containers and external load-balancer software. | - | x |
+| Systemd service management | Kubernetes runs only containers grouped in *pods*. Fleet can manage [several kinds][fleet-unit-types] of [systemd units][systemd-units]. | x | - |
+| Runs rkt or Docker containers | Kubernetes uses the [Docker API][docker-api] to schedule Docker containers on cluster nodes, and uses the standard host-level [systemd-nspawn][systemd-nspawn] to execute rkt containers. Fleet uses [systemd units][systemd-units] to run containers or any other processes on cluster nodes. | x | x |
+| DNS integration | Kubernetes includes a cluster-internal [DNS service][k8s-skydns] for container use. In fleet, such a solution would be constructed from [sidekicks][sidekick] app containers and [service discovery][service-discovery]. | - | x |
+| Graphical user interface (GUI) | The Kubernetes [dashboard][k8s-dashboard] provides a minimal cluster mangement GUI, and commercial solutions like Tectonic elaborate this graphical management. Fleet also has an external [dashboard][fleet-ui] project. | - | x |
+| Schedule jobs depending on system resources | Kubernetes uses the [compute resources][compute-resources] abstraction to decide how to distribute load across the cluster. Fleet simply schedules systemd units. | - | x |
+| Access Control and ACLs | Kubernetes has built-in [roles][k8s-roles] which provide access restrictions. Fleet doesn't have ACLs, and [by design][security] everyone who has access to the etcd cluster can manage fleet units. | - | x |
```

Comment:
Subtle! Because you could say that "roles" is **a** feature, therefore singular, and so the verb should agree as "provide**s**". But in this sentence I think it reads easier for english ears if we write "roles provide..." rather than "roles provides".